pkgs/top-level/python-packages.nix

```
@@ -1890,6 +1890,8 @@ in {
 
   dropbox = callPackage ../development/python-modules/dropbox { };
 
+  ds_store = callPackage ../development/python-modules/ds_store { };
```

Please normalize the attribute name. According to the [Python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names).

```suggestion
  ds-store = callPackage ../development/python-modules/ds-store { };
```

The `pname` can remain as-is.

---

Addressed in 7a3427d